app/controllers/staff/activity_historical_events_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Staff::ActivityHistoricalEventsController < Staff::BaseController
+  include Secured
```

Comment:
I think this is included already in `BaseController` - no biggie, just checking my understanding!

Reply:
yes you are right. I was just copy/pasting from other similar controllers like `Staff::ActivityTransfersController`, `Staff::ActivityChildrenController` etc which also inherit from `Staff::BaseController`...

I see that the inclusion of the `Secured` module is an original feature of the `Staff::BaseController` dating  back to Nov 2019 (https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/commit/679316fe11526f129fd2ca99501b8196efe3dcbd).

It looks like more than half the controllers inheriting from `Staff::BaseController` are performing the same extra include.

So I think a general refactoring on a different PR would be the neatest way to tidy this up generally?